packages/@okta/vuepress-site/docs/concepts/direct-authentication/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Direct authentication
+meta:
+  - name: description
+    content: An overview of what direct authentication is and how you can use it.
+---
+
+# Direct authentication
+
+<!-- Direct authentication simplifies the authentication flow, enhances user experience by minimizing redirects, and provides developers with greater control over security measures. -->
+## What is Direct authentication
+
+[Direct authentication](/docs/guides/configure-direct-auth-grants/aotp/main/) provides a set of [authentication APIs](https://developer.okta.com/docs/api/openapi/okta-oauth/oauth/tag/CustomAS/#tag/CustomAS/operation/tokenCustomAS). These APIs allow a client to directly verify authentication factors for a user in exchange for OAuth 2.0 tokens. The client does this without needing to interact with the Sign-In Widget in a browser.
```

Comment:
I'm wondering if the `authentication APIs` link is confusing. It brings you to the custom AS token endpoint, which is one of the multiple authentication apis it could link to. Could just unlink but I do see why you chose this one

Reply:
I wondered, too. I think I'll take it out.